### ISSUE TITLE
训练部分遮挡数据集与论文中有偏差且较大

### ISSUE THREAD

**Issue by hundunshu** [mangye16/ReID-Survey#13]
您好，我在训练partial_reid数据集时，精度一直达不到论文的要求，相差还是蛮大的，

The test feature is normalized
2021-03-05 16:19:15,585 reid_baseline INFO: Validation Results - Epoch: 110
2021-03-05 16:19:15,585 reid_baseline INFO: mINP: 34.9%
2021-03-05 16:19:15,585 reid_baseline INFO: mAP: 50.2%
2021-03-05 16:19:15,585 reid_baseline INFO: CMC curve, Rank-1  :57.7%
2021-03-05 16:19:15,585 reid_baseline INFO: CMC curve, Rank-3  :66.0%
2021-03-05 16:19:15,585 reid_baseline INFO: CMC curve, Rank-5  :72.7%
2021-03-05 16:19:15,585 reid_baseline INFO: CMC curve, Rank-10 :80.3%
The test feature is normalized
2021-03-05 16:19:16,431 reid_baseline INFO: Validation Results - Epoch: 110
2021-03-05 16:19:16,431 reid_baseline INFO: mINP: 54.8%
2021-03-05 16:19:16,431 reid_baseline INFO: mAP: 54.8%
2021-03-05 16:19:16,431 reid_baseline INFO: CMC curve, Rank-1  :42.9%
2021-03-05 16:19:16,431 reid_baseline INFO: CMC curve, Rank-3  :60.5%
2021-03-05 16:19:16,431 reid_baseline INFO: CMC curve, Rank-5  :65.5%
2021-03-05 16:19:16,431 reid_baseline INFO: CMC curve, Rank-10 :73.9%
配置：
MODEL:
  PRETRAIN_CHOICE: 'imagenet'
  PRETRAIN_PATH: '../resnet50-19c8e357.pth'
  CENTER_LOSS: 'on'
  CENTER_FEAT_DIM: 2048
  NAME: 'resnet50_nl'
  WEIGHT_REGULARIZED_TRIPLET: 'on'
  GENERALIZED_MEAN_POOL: 'on'

INPUT:
  IMG_SIZE: [256, 128]
  PROB: 0.5 # random horizontal flip
  RE_PROB: 0.5 # random erasing
  PADDING: 10

DATASETS:
  NAMES: ('market1501')

DATALOADER:
  PK_SAMPLER: 'on'
  NUM_INSTANCE: 4
  NUM_WORKERS: 8

SOLVER:
  OPTIMIZER_NAME: 'Adam'
  MAX_EPOCHS: 120
  BASE_LR: 0.00035

  CENTER_LR: 0.5
  CENTER_LOSS_WEIGHT: 0.0005

  WEIGHT_DECAY: 0.0005
  IMS_PER_BATCH: 64

  STEPS: [40, 70]
  GAMMA: 0.1

  WARMUP_FACTOR: 0.01
  WARMUP_ITERS: 10
  WARMUP_METHOD: 'linear'

  CHECKPOINT_PERIOD: 40
  LOG_PERIOD: 20
  EVAL_PERIOD: 10

TEST:
  IMS_PER_BATCH: 128
  RE_RANKING: 'off'
  WEIGHT: "path"
  FEAT_NORM: 'on'
  EVALUATE_ONLY: 'off'
  PARTIAL_REID: 'on'

OUTPUT_DIR: "../log/market1501/Experiment-AGW-baseline"

_C.DATASETS.ROOT_DIR = ('../dataset')

dataset文件夹下面有market1501数据集、partial_reid数据集、partial_ilids数据集

数据集是在给出的链接上下载得到的

请教大佬，这个是由于什么原因造成的呀？非常感谢！

**Comment by Hellomodo** [comment]
应该是pytorch和torchvision的版本没对上，确保这两个库的版本如下：
pytorch=1.0.0
torchvision=0.2.1
